app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: false }, html: { class: "sign-up"}) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
```

Comment:
- [ ] Kindly ensure that your design for the sin up page matches the given wireframe as this is a requirement

Your design
![image](https://user-images.githubusercontent.com/81932478/221381185-0e6779d4-6f57-4f61-b832-d432a692fa19.png)

Given wireframe
![image](https://user-images.githubusercontent.com/81932478/221381207-202e9bf0-daa0-4ea4-8cb6-efd828092eca.png)

Reply:
@munsa1, Did you notice that the wireframe you're referring to is a log in page and not a sign up?